Python/peephole.c

```diff
@@ -353,11 +365,16 @@ PyCode_Optimize(PyObject *code, PyObject* consts, PyObject *names,
                            stack effect */
                         h = set_arg(codestr, i, get_arg(codestr, tgt));
                     } else {
+                        Py_ssize_t arg = (tgt + 1);
+                        /* cannot overflow: codelen <= INT_MAX */
+                        assert((size_t)arg <= UINT_MAX / sizeof(_Py_CODEUNIT));
+                        arg *= sizeof(_Py_CODEUNIT);
+
                         /* The second jump is not taken if the first is (so
```

Comment:
I would keep the comment at the beginning of the block.

Reply:
done